frontend/src/app/components/observers/observers.component.ts

```diff
@@ -0,0 +1,43 @@
+import { ObserversStateItem } from '../../store/observers/observers.state';
+import { AppState } from '../../store/store.module';
+import { Store } from '@ngrx/store';
+import { LabelValueModel } from '../../models/labelValue.model';
+import { ApiService } from '../../core/apiService/api.service';
+import { Observable, Subscription } from 'rxjs/Rx';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import * as _ from 'lodash';
+
+@Component({
+  selector: 'app-observers',
+  templateUrl: './observers.component.html',
+  styleUrls: ['./observers.component.scss']
+})
+export class ObserversComponent implements OnInit, OnDestroy {
+
+  observersState: ObserversStateItem[];
+  sub: Subscription;
+
+  anyObservers = false;
+
+  constructor(private http: ApiService, private store: Store<AppState>) { }
+
+  
+  canShowItem(item: ObserversStateItem){
+    return item && !item.error && !item.loading && item.values && item.values.length;
+  }
+
+  ngOnInit() {
+      this.sub = this.store
+        .select(state => state.statistics)
```

Comment:
I'd personally use proper selectors here. One big advantage is that you don't actually need to subscribe to the observable and you can just `| async` it in the template. Also, you're using an older version of RxJS, nowadays the rx operators are used in a `.pipe()`.

Reply:
Learning Angular is WIP as well :)